cli/mender-artifact/main.go

```diff
@@ -405,6 +405,10 @@ func getCliContext() *cli.App {
 			Name:  "mode, m",
 			Usage: "Set the permission bits in the file",
 		},
+		cli.BoolFlag{
+			Name:  "directory, d",
+			Usage: "Install a directory into an artifact",
```

Comment:
I would rephrase as `Create a directory inside an artifact`.

Reply:
Fixed the phrasing.